js/main.js

```diff
@@ -1,28 +1,57 @@
 var entries = $("#entries");
-
 var form = $("form");
 var inputBox = $("#newEntry");
-
 var totalEl = $("#total");
 var total = 0.00;
 
 form.on("submit", updateRegister);
 
-function updateRegister() {
-  event.preventDefault();
-
-  // get the amount and clear the box
-  var amount = parseFloat(inputBox.val());
-  inputBox.val("");
-
-  // make the new element and add it to the list of entries
-  var newEl = $("<tr>");
-  newEl.append("<td></td>");
-  newEl.append("<td>" + "$" + amount.toFixed(2) + "</td>");
-
-  entries.append(newEl);
-
-  // update the total
-  total = total + amount;
-  totalEl.text("$" + total.toFixed(2));
+function Register() {
+  this.domElement = null;
+  this.total = 0.00;
+  this.initialize = function(container){
+    this.domElement = $('<div class="register"></div>');
+    container.append(this.domElement);
+  };
+  this.render = function(){
+    var registerFrontEnd =  '<div class="ticket">'
+    '<h1>Thank You!</h1>' +
+    '<table>' +
+    '<tbody class="entries">' +
+    '</tbody>' +
+    '<tfoot>' +
+    '<tr>' +
+    '<th>Total</th>' +
+    '<th class="total">$0.00</th>' +
+    '</tr> '+
+    '</tfoot>' +
+    '</table>' +
+    '</div>'+
+    '<form class="entry">' +
+    '<input class="newEntry" autofocus placeholder="How Much?">' +
+    '</form>';
+    this.domElement.html = registerFrontEnd
+  }
+
+  this.updateRegister = function () {
+    event.preventDefault();
```

Comment:
This `event.preventDefault();` isn't doing anything because it needs to be connected to an element through an event listener. Without an event listener we also don't know when to call the update function.
```js
// in this.render
form = this.domElement.find(".entry")
form.on("submit", this.updateRegister().bind(this));
```